terraform/aws/modules/efs/efs.tf

```diff
@@ -0,0 +1,11 @@
+/*This will create EFS in bursting mode.The mode can be changed as per the requirement*/
+resource "aws_efs_file_system" "elastic_file_system" {
+  creation_token = var.creation_token
+
+  lifecycle_policy {
+    transition_to_ia = "AFTER_30_DAYS"
+  }
+  performance_mode = var.performance_mode
+  throughput_mode  = var.throughput_mode
+}
+
```

Comment:
Please add an optional EFS mount id as well.

Reply:
@qburst-praven Created a mount target and terraform plan is updated in the PR comment